src/Cli/DabCliParserErrorHandler.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using CommandLine;
+
+namespace Cli
+{
+    /// <summary>
+    /// Processes errors that occur during parsing of CLI verbs (start, init, export, add, update, etc) and their arguments.
+    /// </summary>
+    public class DabCliParserErrorHandler
+    {
+        /// <summary>
+        /// Processes errors accumulated by each parser in parser.ParseArguments<parsers>().
+        /// For DAB CLI, this only includes scenarios where the user provides invalid DAB CLI input.
+        /// e.g. incorrectly formed or missing options and parameters.
+        /// Additionally, an error is tracked if the user uses an unsupported CLI verb or uses --help.
```

Comment:
```suggestion
        /// Additionally, an error is tracked if the user uses an unsupported CLI verb or uses --help or --version.
```

Just out of curiosity, why are we tracking errors for invoking help or version?